.gitignore

```diff
@@ -4,6 +4,7 @@
 *.swp
 *~
 /examples/*/node_modules/
+/examples/mongodb/globalConfig.json
```

Comment:
Or should the example recommend putting it in `os.tmpdir()` so that even canceling mid run does not have leftover files in the repo?

Reply:
Although I guess that could cause problems between Jest projects running in parallel, would have to include a project or run identifier